test/tmp.py

```diff
@@ -0,0 +1,26 @@
+import sys
```

Comment:
Again, this 'tmp' file looks suspicious.

Reply:
Yep. Deleted.